astronomer/providers/google/cloud/triggers/wait_container.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, Dict, Optional, Tuple
+
+from astronomer.providers.cncf.kubernetes.triggers.wait_container import (
+    WaitContainerTrigger,
+)
+
+
+class GKEWaitContainerTrigger(WaitContainerTrigger):
+    """Wrapper for GKE"""
```

Comment:
can you add the params of WaitContainerTrigger doc string in this child class as well